alphazero/model.py

```diff
@@ -0,0 +1,101 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class ResidualBlock(nn.Module):
+    def __init__(self, in_channels, out_channels):
+        super(ResidualBlock, self).__init__()
+        self.conv = nn.Conv2d(in_channels, out_channels, kernel_size=3, stride=1)
+        self.batch_norm = nn.BatchNorm2d(out_channels)
+        self.relu = nn.ReLU(inplace=true)
+
+        def forward(self, x):
+            """
+			Create One Residual Block where we save the input as the residual and add it after two convolutions
+			"""
+            residual = x
+
+            """
+			first convolution
+			"""
+            out = self.conv(x)
+            out = self.batch_norm(out)
+            out = self.relu(out)
+
+            """
+			second convolution, add input after batchnorm step
+			"""
+            out = self.conv(out)
```

Comment:
I'm not super sure, but I think if you use the same `Conv2d` here you get shared weights across two convolutions. I don't think we want that - I think we want one convolution to have separate weights from the other conv.